bin/jobemissions

```diff
@@ -26,6 +27,28 @@ def round_to_nearest_half_hour(t: datetime) -> datetime:
    return t.replace(second=0, microsecond=0)
 
 CI_DATA_DIR = "/work/z19/shared/carbonintensity"
+SCOPE2_AND_3_EMISSIONS_DIR = "/Users/nattybatty/HPC_Emissions_project/emissions-tools/scripts/cleaned_nodes_filled_embodied.csv"
+
+import pandas as pd
+
+# Define the estimated average carbon value
+ESTIMATE_VALUE = 10343.0727848101
+
+df = pd.read_csv(SCOPE2_AND_3_EMISSIONS_DIR)
+
+node_carbon = {
+    row["node_name"]: {
+        "cpu_cores": row["cpu_cores"],
+        "embodied": row["embodied_carbon_kgco2e"],
+        "embodied_estimate": round(row["embodied_carbon_kgco2e"], 6) == round(ESTIMATE_VALUE, 6),
+        "max_power_draw": row["energy_100pct_kw"],
+        "min_power_draw": row["energy_0pct_kw"],
+        "model_id": row["model_manufacturer_id"]
+    }
+    for _, row in df.iterrows()
+}
```

Comment:
We can use the `read_node_data` function you wrote here - that way we make sure that this script is using the tested function